src/@components/detailPage/orderContents.tsx

```diff
@@ -0,0 +1,89 @@
+import { PRODUCT_DATA } from "../../core/productData";
+import { OrderOptionProps } from "../../type/OrderOptionType";
+import { MinusIc } from "../../assets";
+import { PlusIc } from "../../assets";
+import { styled } from "styled-components";
+
+export default function OrderContents({ discountedPrice }: OrderOptionProps) {
```

Comment:
```suggestion
export default function OrderContents(props: OrderOptionProps) {
    const { discountedPrice } = props;
```
요렇게 구조 분해 할당해보는 것은 어떨까용? 나중에 props가 엄청 나게 많아진다면, 컴포넌트 함수 옆에 썼을 때 너무 길어질 수 있으니까 props로 받고, props를 구조 분해하는 방법도 좋을 것 같아용~